gax-java/gax/src/test/java/com/google/api/gax/core/GaxPropertiesTest.java

```diff
@@ -175,6 +174,52 @@ public void testGetBundleVersion_noManifestFile() throws IOException {
     assertFalse(version.isPresent());
   }
 
+  @Test
+  void testGetProtobufVersion_success() throws Exception {
+    IClassLoaderWrapper mockClassLoader = mock(IClassLoaderWrapper.class);
```

Comment:
Just reiterating while looking at this logic: The interface is not necessary to be able to mock the wrapper class.